build.zig.zon

```diff
@@ -0,0 +1,10 @@
+.{
+    .name = "tree-sitter",
+    .version = "0.22.5",
```

Comment:
Note: another manifest file that needs to be bumped on release; should be added to whatever workflow you have for that @amaanq